BurnOutSharp/ProtectionType/AACS.cs

```diff
@@ -12,23 +12,61 @@ public string CheckPath(string path, bool isDirectory, IEnumerable<string> files
         {
             if (isDirectory)
             {
-                if (files.Any(f => f.Contains(Path.Combine("aacs", "VTKF000.AACS")))
-                    && files.Any(f => f.Contains(Path.Combine("AACS", "CPSUnit00001.cci"))))
+                if (files.Any(f => f.Contains(Path.Combine("AACS", "MKBROM.AACS"))))
                 {
-                    return "AACS";
+                    string versionPathHDDVD = files.FirstOrDefault(f => Path.GetFileName(f).Equals("MKBROM.AACS", StringComparison.OrdinalIgnoreCase));
+                    int? version = GetVersion(versionPathHDDVD);
+                    if (version == null)
+                        return "AACS (Unknown Version)";
+                    return $"AACS Version {version}";
+                }
+                if (files.Any(f => f.Contains(Path.Combine("AACS", "MKB_RO.inf"))))
+                {
+                    string versionPathBD = files.FirstOrDefault(f => Path.GetFileName(f).Equals("MKB_RO.inf", StringComparison.OrdinalIgnoreCase));
+                    int? version = GetVersion(versionPathBD);
+                    if (version == null)
+                        return "AACS (Unknown Version)";
+                    return $"AACS Version {version}";
                 }
             }
             else
             {
                 string filename = Path.GetFileName(path);
-                if (filename.Equals("VTKF000.AACS", StringComparison.OrdinalIgnoreCase)
-                    || filename.Equals("CPSUnit00001.cci", StringComparison.OrdinalIgnoreCase))
+                if (filename.Equals("MKBROM.AACS", StringComparison.OrdinalIgnoreCase))
                 {
-                    return "AACS";
+                    string versionPathHDDVD = path;
+                    int? version = GetVersion(versionPathHDDVD);
+                    if (version == null)
+                        return "AACS (Unknown Version)";
+                    return $"AACS Version {version}";
                 }
+                if (filename.Equals("MKB_RO.inf", StringComparison.OrdinalIgnoreCase))
+                {
+                    string versionPathBD = path;
+                    int? version = GetVersion(versionPathBD);
+                    if (version == null)
+                        return "AACS (Unknown Version)";
+                    return $"AACS Version {version}";
+                } 
             }
 
             return null;
         }
+        private static int? GetVersion(string path)
+        {
+            try
```

Comment:
Forgot to comment this the last time, but please put a file existence check before the `try` so you can return `null` without having to throw an exception.